docs/sdk-upgrade.md

```diff
@@ -0,0 +1,154 @@
+
+10. After you have photographed the front and back of your card, you have tested the SDK locally. You do not need to submit the images. If you have both an Android and an iPhone, test the SDK with both of them. (Or, pair with someone who has the other type of phone.)
+
+11. Open a pull request for the modified SDK files. The next process &mdash; A/B testing the new version &mdash; can only take place after the files are accepted into the main branch.
+
+## Turn on A/B testing
+
+After you have added the new SDK files per [the above instructions](#add-new-sdk-files), and after those new files have been merged into the main Git branch and deployed, you may A/B test the new Acuant SDK version.
+
+You will need:
+* the AWS prod-power role ([request access](https://github.com/18F/identity-devops/issues/new?assignees=&labels=administration&template=onboarding-devops-prod.md&title=Onboarding+to+Production+for+%5BTEAM_MEMBER%5D))
+* a YubiKey ([setup](https://github.com/18F/identity-devops/wiki/Setting-Up-your-Login.gov-Infrastructure-Configuration#configuring-a-yubikey-as-a-virtual-mfa-device))
+* a GFE computer
+
+1. In identity-devops, run the command:
+
+    ```zsh
+    aws-vault exec prod-power -- bin/app-s3-secret --env prod --app idp --edit
+    ```
+
+    The command downloads a configuration file and opens an editor so you can modify it.
+
+2. Make the file look like this:
+
+    ```yml
+    idv_acuant_sdk_upgrade_a_b_testing_enabled: true
+    idv_acuant_sdk_upgrade_a_b_testing_percent: 50
+    idv_acuant_sdk_version_alternate: 11.M.M
+    idv_acuant_sdk_version_default: 11.N.N
+    ```
+ 
+    Set the default to the new SDK version and the alternate to the old version. (That way, the new version is in place if the A/B testing goes well.) The percentage should already be set to 50, so it does not need to be changed.
+    
+3. Save the file. If the file opened in the vi editor, use `:wq` to save. A diff of your changes will appear. Type `y` to accept them.
+
+4. Recycle the servers to deploy the changes:
+
+    ```zsh
+    aws-vault exec prod-power -- bin/asg-recycle prod idp
+    ```
+    
+    The above uses [this script](https://github.com/18F/identity-devops/wiki/Deploying-Infrastructure-Code#recycle-hosts) to kick off a migration instance, run database migrations, and double the instances into a 50/50 state.
+
+Monitoring the A/B test begins now. Proceed to the next section.
+
+## Monitor A/B testing
+
+#### For 15 minutes after deploy:
+
+Use our [`ls-servers` command](https://handbook.login.gov/articles/devops-scripts.html#ls-servers) to view a table of our servers:
+
+```zsh
+aws-vault exec prod-power — bin/ls-servers -e prod
+```
+
+Monitor that the servers come back up after recycling. Check that they enter the `running` state and accumulate uptime.
```

Comment:
Thanks! Feel free to check again that I got it right

Reply:
Looks better! I wonder if we want to more generally refer people to the production deployment instructions rather than detailing them here, since they might change in the future.

Reply:
Great call; done